index.js

```diff
@@ -8,32 +8,48 @@ var Reader = module.exports = function(options) {
   options = options || {}
   this.offset = 0
   this.lastChunk = false
-  this.chunk = null
+  this.chunk = Buffer.alloc(4);
```

Comment:
currently node-postgres supports node `@v0.10.x` onward - does this exist in older versions of node?

note to self: I need to set up travis on this repo.